scripts/build.js

```diff
@@ -43,10 +50,10 @@ function parse(input) {
 }
 
 function generateLookupFunction(categories) {
-	let code = '// Generated code.\n\nexport default function lookup(x) {\n';
+	const branches = [];
 
 	for (const [category, ranges] of categories) {
-		if (category === DEFAULT_CATEGORY) {
+		if (category === DEFAULT_CATEGORY || ranges.length === 0) {
```

Comment:
I don't think there can be empty ranges? This may be better as an assertion in `parse`.

Reply:
Since I changed the `categories` to 

```js
const categories = new Map(Array.from(CATEGORY_NAMES, ([, category]) => [category, []]));
```

In theory, it's possible if a new "eastAsianWidthProperty" added, but ranges not present/removed.

Reply:
That should definitely be an assertion, so we will know to add the new category.